chsdi/templates/htmlpopup/hartsteinvorkommen.mako

```diff
@@ -94,7 +94,7 @@
     </tr>
     <tr>
       <td class="cell-left">${_(f'{layerid}.elevation')}</td>
-      <td>${c['attributes']['elevation'] or '-'}</td>
+      <td>${int(round(c['attributes']['elevation'])) or '-'}</td>
```

Comment:
I did not test this explicitly here but I would guess that this gives an error if `c['attributes']['elevation']` is `None`.

Here my alternative suggestion:
```
int(round(c['attributes']['elevation'])) if c['attributes']['elevation'] is not None else '-'
```

Reply:
thank you :+1: 
i have changed it